packages/now-cli/test/helpers/prepare.js

```diff
@@ -477,6 +477,15 @@ CMD ["node", "index.js"]`,
         },
       }),
     },
+    'project-link': {
+      'pages/index.js':
+        'export default () => <div><h1>Now CLI test</h1><p>Zero-config + Next.js</p></div>',
```

Comment:
Why does this say Next.js but the dependency is gatsby?

Reply:
```suggestion
        'export default () => <div><h1>Now CLI test</h1></div>',
```